_data/sites.json

```
@@ -8198,6 +8198,18 @@
             "points.com"
         ]
     },
+    
+    {
+        "name": "Pokémon Trainer Club",
+        "url": "https://support.pokemon.com/hc/en-us/articles/360000965826-How-do-I-delete-my-Pok%C3%A9mon-Trainer-Club-account-",
+        "difficulty": "easy",
+        "notes": "Go to https://club.pokemon.com/us/pokemon-trainer-club/edit-profile/ scroll down to the bottom abd click 'Request Data Deletion', enter the required fields and you'll then get a confirmation email.",
```

```suggestion
        "notes": "Go to https://club.pokemon.com/us/pokemon-trainer-club/edit-profile/ scroll down to the bottom and click 'Request Data Deletion', enter the required fields and you'll then get a confirmation email.",
```
Is it deleted immediately?

---

> Is it deleted immediately?

It's deleted when you click the button in the confirmation email

> Click the Delete Account button in the email. A confirmation page will display confirming that you have successfully deleted your Pokémon Trainer Club account.